src/ZMQ.jl

```diff
@@ -4,7 +4,7 @@ VERSION >= v"0.4.0-dev+6521" && __precompile__(true)
 
 module ZMQ
 using Compat
-import Compat.String
+import Compat: String, unsafe_string
```

Comment:
when `import Compat: unsafe_string`, I got
```
julia> using ZMQ
INFO: Recompiling stale cache file /Users/wookyoung/.julia/lib/v0.5/ZMQ.ji for module ZMQ.

julia> method_exists(unsafe_string, (Message,))
true
```

but without `import Compat: unsafe_string`, got
```
julia> using ZMQ
INFO: Recompiling stale cache file /Users/wookyoung/.julia/lib/v0.5/ZMQ.ji for module ZMQ.

julia> method_exists(unsafe_string, (Message,))
false
```

and test has also failed,
```
Testing with ZMQ version 3.2.5
ERROR: LoadError: AssertionError: typeof(ex) == StateError
 in include_from_node1(::String) at ./loading.jl:426
 in process_options(::Base.JLOptions) at ./client.jl:262
 in _start() at ./client.jl:318
while loading /Users/wookyoung/.julia/v0.5/ZMQ/test/runtests.jl, in expression starting on line 16
```